va/gsb.go

```diff
@@ -24,28 +26,33 @@ type SafeBrowsing interface {
 // third-party safe browsing API. It's meant be called by the RA before pending
 // authorization creation. If no third-party client was provided, it fails open
 // and increments a Skips metric.
-func (va *ValidationAuthorityImpl) IsSafeDomain(ctx context.Context, req *core.IsSafeDomainRequest) (*core.IsSafeDomainResponse, error) {
+func (va *ValidationAuthorityImpl) IsSafeDomain(ctx context.Context, req *vaPB.IsSafeDomainRequest) (*vaPB.IsDomainSafe, error) {
```

Comment:
This will cause errors if a passed an old format message from AMQP during a deployment (i.e. if a old version of the RA is up when the new VA is up, or if the AMQP queue contains messages from the old RA when the new VA is started), instead the moving between `core` and `vaPB` versions of `IsSafeDomainRequest` should be handled in the respective wrappers until we are able to completely remove the AMQP layer.